go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openfga/openfga
 
-go 1.21.5
+go 1.22.0
```

Comment:
Can you please keep this at the latest go1.21 version? Currently `go1.21.7`

Reply:
I have updated to go 1.21.7